examples/fastapi/poetry.lock

```diff
@@ -35,23 +35,31 @@ trio = ["trio (>=0.26.1)"]
 
 [[package]]
```

Comment:
should we just .gitignore the lock file?

Reply:
I talked with Sam about this before. We ignore the lockfile in the SDK project(s), but not in these example apps. The thinking is, the examples are intended to be ready-to-run and in that case you do want specific versions of everything "pinned".

Reply:
Makes sense!